### ISSUE TITLE
LDAP error: Error : User could not be authentificated

### ISSUE THREAD

**Issue by nikpael** [nilsteampassnet/TeamPass#3831]
### Steps to reproduce
1. update to 3.0.10

### Expected behaviour
ldap user login

### Actual behaviour
LDAP error: Error : User could not be authentificated

### Server configuration
**Operating system**:
ubuntu 22.04.3 LTS

**Web server:**
 Apache/2.4.52
**Database:**
mysql 8.0.34

**PHP version:**
PHP 8.2.9 (cli) 

**Teampass version:** 
3.0.10

**Teampass configuration file:**

**Updated from an older Teampass or fresh install:**
PLEASE attach to this issue the file `/includes/config/tp.config.php`.

### Client configuration
**Browser:**
chrome last version

#### Web server error log
```
no
```

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
Insert the log here and especially the answer of the query that failed.
```

**Comment by hitenmandalia** [comment]
Im getting the same.  can sync LDAP ok, add the user and the task is fully complete.  However, when trying to log in, im always getting LDAP user could not be authenticated error.

**Comment by nikpael** [comment]
Yes for me it’s the same. The problem is also when i test in ldap tab admin.

**Comment by nikpael** [comment]
Hi, does anyone have any workarounds?

**Comment by nilsteampassnet** [comment]
@nikpael 
Would it be possible to have a screen sharing session?
If yes, contact me on nils@teampass.net

**Comment by hitenmandalia** [comment]
@nilsteampassnet 
If @nikpael is not available, i can also do a screen share at a suitable time

**Comment by nikpael** [comment]
I downgraded TeamPass. Can you do the remote session with @hitenmandalia ? Or if you want I can upgrade and then contact you..

**Comment by nilsteampassnet** [comment]
> @nilsteampassnet If @nikpael is not available, i can also do a screen share at a suitable time

Yes please contact me by mail

**Comment by hitenmandalia** [comment]
@nilsteampassnet email sent

**Comment by nikpael** [comment]
Good morning, do you have news about the bug? Do you need my session? Thanks for your your work

**Comment by jonildoasantos** [comment]
I have the same issue when upgrade from 3.0.6 to 3.0.10

**Steps to reproduce**
update to 3.0.10

**Expected behaviour**
LDAP user login

**Actual behaviour**
LDAP error: Error : User could not be authentificated

**Server configuration**

Operating System:
Ubuntu 22.04.3 LTS

Web Server:
Apache/2.4.52

Database:
mariadb 10.7.8

PHP version:
PHP 8.1.2

TeamPass version:
3.0.10

LDAP Server:
Active Directory Domain Server (2022)

**Comment by gapaalfredo** [comment]
I have the same error on a new installation. can sync LDAP ok, add the user and the task is fully complete. However, when trying to log in, im always getting LDAP user could not be authenticated error.

**Comment by BTa7BxrHYn** [comment]
I have the same error on a new installation.

**Comment by jonildoasantos** [comment]
I try new migration from 3.0.6 to 3.0.10 and work for me. Thanks!

**Comment by nilsteampassnet** [comment]
Closed automatically due to inactivity (no response for over 1 year).
